### DISCUSSION TITLE
auth/users table : couldn't fetch - Nhost

### DISCUSSION THREAD

**Discussion by shyamilitc** [refinedev/refine#1886] [General]
Hi
I am using nhost as my data provider. When I am fetching auth/users table, Its throwing user_aggregate not found in query root error.
I know its because for users table its userAggregate.
export interface IUser {
  id: string;
  displayName: string;
  createdAt: string;
  createdBy: string;
  updatedAt: string;
  updatedBy: string;
}
this is my interface. 
My question is how can i fetch details from users table using refine?

**Comment by omeraplak** [comment]
Hey @shyamilitc ,
Thank you for contacting us. You can use `resource` property of data, form hooks.

https://refine.dev/docs/faq/#how-can-i-request-an-api-with-nested-route

I hope this helped you 🚀

**Comment by shyamilitc** [comment]
Thank you for your response.I have tried tha, The issue is with userAggragate function. Refine is taking for any resource ,resource_aggregate but for user its userAggregate. So I will get user_aggregate not found in query_root error.10.06.2022, 17:56, "Ömer Faruk APLAK" ***@***.***>:
Hey @shyamilitc ,
Thank you for contacting us. You can use resource property of data, form hooks.
https://refine.dev/docs/faq/#how-can-i-request-an-api-with-nested-route
I hope this helped you 🚀

**Reply by omeraplak** [reply]
ah okay. it seems this line is blocking you,
https://github.com/pankod/refine/blob/next/packages/nhost/src/index.ts#L193

We shouldn't add the prefix `_aggragate` if `operation` is passed here. Does this work for you?
